src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -109,7 +110,7 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
   }
 
   onMutationError(errors, errorModalTitle?, errorModalMessage?) {
-    logger.error(errors)
+    logger.error(new ErrorWithMetadata(errors.message || errors.data, errors))
```

Comment:
One snag here is that it's difficult to tell what the "message" is when we get back an error payload from Metaphysics. Sometimes we get back a specific message, and other times we get back stringified JSON data that might contain an error message. This can make for some noisy error messages in the latter case since that object might contain a lot of key/value pairs.

Reply:
Isn’t that dependent on the call-site? If so, I think you should create the error there instead.

Reply:
Yes! You're right, and since we know the type of the error this problem takes care of itself. I'll update that.